UefiPayloadPkg/Library/BaseSerialPortLibHob/BaseSerialPortLibHob.inf

```diff
@@ -18,6 +18,7 @@
 [Packages]
   MdePkg/MdePkg.dec
   MdeModulePkg/MdeModulePkg.dec
+  UefiPayloadPkg/UefiPayloadPkg.dec

```

Comment:
Assuming PcdHandOffFdtEnable is dropped, is this still needed?

Reply:
We expect it will take some time to transition from HOB input to FDT thus providing this PCD for supporting Hob only scenario (to remove FDT complexity and size impact). We may clean this up later once we collected feedbacks and got no concerns from every platforms.

Reply:
I think Benjamin was referring to that this PcdHandOffFdtEnable is not used in BaseSerialPortLibHobinf specifically, so its more on clean up rather than real issue i guess ;)

Reply:
Yeah, I was only thinking of clean up in this file

Reply:
Done.